packages/schematics/angular/utility/json-utils.ts

```diff
@@ -191,3 +191,24 @@ export function findPropertyInAstObject(
 function _buildIndent(count: number): string {
   return count ? '\n' + ' '.repeat(count) : '';
 }
+
+function _strigifyContent(value: JsonValue, indentStr: string): string {
+  // TODO: Add snapshot tests
```

Comment:
What do you mean by this?

Reply:
We should add golden files/snapshots tests for json ast manipulation to confirm that the indentation inserted is correct when modifying a file.

Currently it’s really hard to understand if the indentation is correct and also test the indentations itself.

Haven’t golden files/snapshot tests will be more straightforward but Ofc we need to set this up.

Reply:
That function is spelled wrong btw.

We should really drop these when working with the workspace but we'll need to fix migrations first so that we can use the latest APIs inside a migration.

Reply:
Oops thanks for pointing that out.